conda/cli/common.py

```diff
@@ -211,3 +211,10 @@ def check_non_admin():
             The create, install, update, and remove operations have been disabled
             on your system for non-privileged users.
         """))
+
+def is_valid_prefix(prefix):
+    if isdir(prefix):
+        if not isfile(join(prefix, 'conda-meta', 'history')):
```

Comment:
Should this be documented somewhere that conda requires this file at all times?

Reply:
Forrest and I discussed it and decided this isn't something that the majority of our users would need to know except in some edge cases so to avoid confusing folks, we're not going to add it to docs. Thanks for thinking of the doc impact, though!

Reply:
ya, the existence of that file is an indication that the prefix is indeed a valid conda environment. It's more of an implementation detail I think

Reply:
I really meant documented internally, so someone doesn't change the name of that file in a later implementation.